src/box/flightrec.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2022, Tarantool AUTHORS, please see AUTHORS file.
+ */
+#include "flightrec.h"
+
+#include "say.h"
+#include "trivia/config.h"
+
+#if defined(ENABLE_FLIGHT_RECORDER)
+# error unimplemented
+#endif
+
+void
+flightrec_init(const char *fr_dirname, size_t logs_size,
+	       size_t logs_max_msg_size, int logs_log_level)
+{
+	(void)fr_dirname;
+	(void)logs_size;
+	(void)logs_max_msg_size;
+	(void)logs_log_level;
+	say_error("Flight recorder is not available in this build");
+}
+
+void
+flightrec_destroy(void)
```

Comment:
Should be named `flightrec_free` in accordance with our naming conventions.

Reply:
Fixed.